manifests/init.pp

```diff
@@ -258,7 +258,6 @@
   # Anchor these separately so as not to break
   # the notify between main classes
   Class['foreman::install'] ~>
-  class { '::foreman::compute': } ~>
```

Comment:
I don't think this block does anything right now, but I think we ought to keep the idea going so that any installation of a foreman-vmware package etc notifies the foreman::service class.

Maybe we could add `tag => 'foreman-compute'` to all of those package resources inside the CR classes and then do:

<pre>
Class['foreman::install'] ~>
Package <| tag == 'foreman-compute' |> ~>
Class['foreman::service']
</pre>